lib/ts/Regex.cc

```diff
@@ -136,6 +136,12 @@ Regex::~Regex()
   }
 }
 
+void
+Regex::operator delete(void *ptr)
+{
+  ::free(ptr);
```

Comment:
Hmmm, this shouldn't be its_free() ?

Reply:
The keyword `delete` first invokes the destructor, in this case `Regex::~Regex` then calls `Regex::operator delete` (if it exists) to clean up the memory. Testing indicates that calling the destructor here breaks things.

Reply:
But we should still use ats_free(), no ?